examples/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "acuvity-ts-examples",
+  "version": "1.0.0",
+  "main": "basic_text.js",
+  "type": "module",
+  "scripts": {
+    "test": "node basic_file.js && node basic_text.js"
```

Comment:
I would break these into separate script tasks for all the examples that we have. Similarly, how for deno you can do this with the tasks in the `deno.json` config.